gmplot/gmplot.py

```diff
@@ -64,9 +68,9 @@ def scatter(self, lats, lngs, color=None, size=None, marker=True, c=None, s=None
             else:
                 self.circle(lat, lng, size, **settings)
 
-    def circle(self, lat, lng, radius, color=None, c=None, **kwargs):
+    def circle(self, lat, lng, radius, color=None, c=None, alpha=0.5, **kwargs):
         color = color or c
-        kwargs.setdefault('face_alpha', 0.5)
+        kwargs.setdefault('face_alpha', alpha if alpha>0 else 0.5)
```

Comment:
I'd say we should allow users to provide any value of `alpha` here, so `kwargs.setdefault('face_alpha', alpha)` should be fine (in case you ever want a transparent circle with only a visible edge).